packages/commonwealth/scripts/refresh-all-memberships.ts

```diff
@@ -37,8 +41,16 @@ async function main() {
   for (const community of communities) {
     if (process.env.COMMUNITY_ID && process.env.COMMUNITY_ID !== community.id)
       continue;
-    await groupsController.refreshCommunityMemberships({
-      communityId: community.id,
+
+    const admin = await models.Address.findOne({
+      where: { community_id: community.id, role: 'admin' },
+    });
```

Comment:
I'm getting an error locally here because no admin is found. Could be data corruption on my end, but nonetheless there's no use in using a real entity here because this script is only ever executed manually by a superadmin.

Reply:
Yep, all communities should have one admin... this is the new auth requirement of the refresh command... I tried with super admin but it also has to be a member of the community, so looks like the community admin is a safer option.... guess we can ignore errors like this and not refresh groups when admin is corrupted